docs/trino-connector/development.md

```diff
@@ -77,142 +72,75 @@ then you can see the `gravitino-trino-connecor` source files and directories in
             <groupId>io.airlift</groupId>
             <artifactId>json</artifactId>
         </dependency>
 
         <dependency>
             <groupId>io.trino</groupId>
-            <artifactId>trino-plugin-toolkit</artifactId>
+            <artifactId>trino-spi</artifactId>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
-            <groupId>jakarta.validation</groupId>
-            <artifactId>jakarta.validation-api</artifactId>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-jdbc</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-collections4</artifactId>
-            <version>4.4</version>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-client</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.httpcomponents.client5</groupId>
-            <artifactId>httpclient5</artifactId>
-            <version>5.2.1</version>
-        </dependency>
-
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
 
         <dependency>
-            <groupId>io.airlift</groupId>
-            <artifactId>slice</artifactId>
-            <scope>provided</scope>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-jdbc</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-api</artifactId>
-            <scope>provided</scope>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.4</version>
         </dependency>
 
         <dependency>
-            <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-context</artifactId>
-            <scope>provided</scope>
+               <groupId>org.slf4j</groupId>
+               <artifactId>slf4j-api</artifactId>
+               <version>1.7.32</version>
         </dependency>
 
         <dependency>
-            <groupId>io.trino</groupId>
-            <artifactId>trino-spi</artifactId>
-            <scope>provided</scope>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>2.0.9</version>
         </dependency>
 
         <dependency>
-            <groupId>org.openjdk.jol</groupId>
-            <artifactId>jol-core</artifactId>
-            <scope>provided</scope>
+            <groupId>com.datastrato.gravitino</groupId>
+            <artifactId>gravitino-client-java-runtime</artifactId>
+            <version>0.6.0-SNAPSHOT</version>
```

Comment:
`0.6.0-SNAPSHOT` has not been release, you may need to change to 0.5.0 and add some comment like 
```
You can switch to the snapshot version as you like,  for example, if you want to latest main branch, you can execute the following command the install Gravitino `gravitino-client-java-runtime` jar locally.
./gradlew publishToMavenLocal

```

Reply:
Additionally, the artifact ID is incorrect. This is the version in the Maven central repository.
<!-- https://mvnrepository.com/artifact/com.datastrato.gravitino/client-java-runtime -->
<dependency>
    <groupId>com.datastrato.gravitino</groupId>
    <artifactId>client-java-runtime</artifactId>
    <version>0.5.0</version>
</dependency>

<img width="1162" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/0e809398-219d-41b6-93ea-15bcddff5914">